doc/cephadm/adoption.rst

```diff
@@ -105,7 +106,8 @@ Adoption process
 
 #. Redeploy MDS daemons by telling cephadm how many daemons to run for
    each file system.  You can list file systems by name with ``ceph fs
-   ls``.  For each file system::
+   ls``.  For each file system. 
+   This needs to be run on the master nodes::
```

Comment:
looks odd. What does `For each file system` mean?

Reply:
I have reworded this in my branch, here:

https://github.com/zdover23/ceph/tree/wip-doc-cephadm-adoption-step-9

I would prefer to abandon this PR, since I can't fix up the Signed-off-by line in this branch.

Reply:
Sure do what you think is best.